test.js

```diff
@@ -33,7 +33,7 @@ describe('artsyXapp', function() {
     });
   });
 
-  it('refreshes the access token before it expires', function(done) {
+  xit('refreshes the access token before it expires', function(done) {
```

Comment:
This test suite is a little weird, the test results bleed into each other (ie- while this test is waiting for a refresh, other tests are running which changes the current token). It would be nice tho to get this working!

Reply:
Please make a follow up ticket to address this!

Reply:
Will do! https://artsyproduct.atlassian.net/browse/PLATFORM-1517